src/components/accessories/patientNewBill/PatientNewBill.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { FC, useCallback, useState } from "react";
+import "./styles.scss";
+import { useTranslation } from "react-i18next";
+import SmallButton from "../smallButton/SmallButton";
+import BillItemsTable from "./itemsTable/BillItemsTable";
+import { NewBillSide } from "./NewBillSide";
+import { CustomModal } from "../customModal/CustomModal";
+import BillItemPickerForm from "./itemPicker/BillItemPicker";
+import { PaymentDialog } from "../paymentDialog/PaymentDialog";
+import { BillPaymentsDTO } from "../../../generated";
+
+const PatientNewBill: FC = () => {
+  const { t } = useTranslation();
+  const [showItemPicker, setShowItemPicker] = useState(false);
+  const [showPaymentDialog, setShowPaymentDialog] = useState(false);
+
+  const handleItemPickerShow = useCallback(() => {
+    setShowItemPicker(false);
+  }, []);
+  const handlePaymentDialogClose = useCallback(() => {
+    setShowPaymentDialog(false);
+  }, []);
+  const handlePayment = useCallback((payment: BillPaymentsDTO) => {}, []);
+
+  return (
+    <>
+      <div className="patientNewBill">
+        <div className="main">
+          <div className="patientNewBill_left">
+            <div>
+              <span className="addpayment hidden">
+                {t("bill.clicktoaddpayment")}
+              </span>
+            </div>
+            <BillItemsTable
+              handleDelete={(row) => {}}
+              handleEdit={(row) => {
+                setShowItemPicker(true);
+              }}
+            />
+            <div className="patientNewBill_buttons">
+              <SmallButton
+                onClick={() => {
+                  setShowItemPicker(true);
+                }}
+              >
+                {t("button.add")}
+              </SmallButton>
+              <SmallButton onClick={() => {}}>{t("button.save")}</SmallButton>{" "}
```

Comment:
Put the Add button to the bottom right of the table (it's just a UI trick, the call to action must stay to the right position).

And then, the put the save button inside the column with the rest of the buttons with the label "Save as draft"